code/game/machinery/computer/account_manager.dm

```diff
@@ -138,6 +138,9 @@
 				if(H.mind.assigned_role == "Captain" && GLOB.using_map.name == "Nerva")
 					continue	//We don't pay the captain
 
+				if(H.mind.assigned_role == "Passenger" && GLOB.using_map.name == "Nerva")
```

Comment:
Passenger job is Nerva only, so this map name check is redundant. I also can't envision passengers needing to be on the payroll for _any_ map
```suggestion
				if(H.mind.assigned_role == "Passenger")
```